scenarios/kubernetes_e2e.py

```diff
@@ -423,7 +452,7 @@ def main(args):
       'KUBE_GKE_NETWORK=%s' % cluster,
     )
 
-    mode.start(runner_args)
+    mode.start(args, runner_args)
```

Comment:
seems this is not necessary, you can call mode.add_environment outside mode.start